couchbase/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   db:
-    image: arungupta/couchbase
+    image: CelsoSantos/couchbase
```

Comment:
Most of the files seems to use that image. Can you update the PR to revert back to `arungupta/couchbase`?